### ISSUE TITLE
error: uncaughtException: Something went wrong. cron reached maximum iterations.

### ISSUE THREAD

**Issue by radosi** [kelektiv/node-cron#607]
Time Zone: "" - Cron String: 0 0,5,10,15,20,25,30,35,40,45,50,55 * * * * - UTC offset: +03:00 - current Date: Tue Apr 26 2022 11:13:48 GMT+0300
    at CronTime._getNextDateFrom (/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js:235:12)
    at CronTime.sendAt (/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js:156:17)
    at CronTime.getTimeout (/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js:175:29)
    at CronJob.start (/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js:613:31)
    at Timeout.callbackWrapper [as _onTimeout] (/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js:665:29)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7) {"error":{},"stack":"Error: Something went wrong. cron reached maximum iterations.\n\t\t\t\t\t\tPlease open an  issue (https://github.com/kelektiv/node-cron/issues/new) and provide the following string\n\t\t\t\t\t\tTime Zone: \"\" - Cron String: 0 0,5,10,15,20,25,30,35,40,45,50,55 * * * * - UTC offset: +03:00 - current Date: Tue Apr 26 2022 11:13:48 GMT+0300\n    at CronTime._getNextDateFrom (/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js:235:12)\n    at CronTime.sendAt (/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js:156:17)\n    at CronTime.getTimeout (/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js:175:29)\n    at CronJob.start (/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js:613:31)\n    at Timeout.callbackWrapper [as _onTimeout] (/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js:665:29)\n    at listOnTimeout (node:internal/timers:559:17)\n    at processTimers (node:internal/timers:502:7)","exception":true,"date":"Tue Apr 26 2022 11:13:48 GMT+0300 (Eastern European Summer Time)","process":{"pid":10623,"uid":1000,"gid":1000,"cwd":"/home/radosi/ppn/admin/backend","execPath":"/home/radosi/.nvm/versions/node/v16.14.2/bin/node","version":"v16.14.2","argv":["/home/radosi/.nvm/versions/node/v16.14.2/bin/node","/home/radosi/ppn/admin/backend/src/app.ts"],"memoryUsage":{"rss":277127168,"heapTotal":196325376,"heapUsed":184823216,"external":1096522,"arrayBuffers":141004}},"os":{"loadavg":[18,18,18],"uptime":90317},"trace":[{"column":12,"file":"/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js","function":"CronTime._getNextDateFrom","line":235,"method":"_getNextDateFrom","native":false},{"column":17,"file":"/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js","function":"CronTime.sendAt","line":156,"method":"sendAt","native":false},{"column":29,"file":"/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js","function":"CronTime.getTimeout","line":175,"method":"getTimeout","native":false},{"column":31,"file":"/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js","function":"CronJob.start","line":613,"method":"start","native":false},{"column":29,"file":"/home/radosi/ppn/admin/backend/node_modules/cron/lib/cron.js","function":"Timeout.callbackWrapper [as _onTimeout]","line":665,"method":"callbackWrapper [as _onTimeout]","native":false},{"column":17,"file":"node:internal/timers","function":"listOnTimeout","line":559,"method":null,"native":false},{"column":7,"file":"node:internal/timers","function":"processTimers","line":502,"method":null,"native":false}]}
[nodemon] app crashed - waiting for file changes before starting...

**Comment by intcreator** [comment]
closing in favor of #467
